snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -97,10 +97,19 @@ public class Snapshot: NSObject {
         print("snapshot: \(name)") // more information about this, check out https://github.com/fastlane/fastlane/tree/master/snapshot
 
         sleep(1) // Waiting for the animation to be finished (kind of)
-        XCUIDevice.sharedDevice().orientation = .Unknown
+
+        #if os(tvOS)
+            XCUIApplication().childrenMatchingType(.Browser).count
```

Comment:
Yep, I noticed it first on the iOS tests as part of the existing `waitForLoadingIndicatorToDisappear()` check: 

```
                         <key>HasScreenshotData</key>
                         <true/>
                         <key>Title</key>
                         <string>Get number of matches for: Children matching type Other</string>
                         <key>UUID</key>
                         <string>CDE58247-BB09-45B1-AF2F-258A4A5EA103</string>
```